vocab/types.json

```diff
@@ -935,31 +935,31 @@
         "translatableTo": null
     },
     "https://openminds.ebrains.eu/core/Comment": {
-        "description": null,
+        "description": "Structured information about a short text associated with some entity.",
```

Comment:
"... short comment associated..." ?

Reply:
using "comment" makes it a bit recursive, no?

I did think about "a short text expressing an opinion on, or giving information about some entity" rather than "associated with"

Reply:
yes I like that